homeassistant/components/switch/lutron_caseta.py

```diff
@@ -0,0 +1,54 @@
+"""Support for Lutron Caseta switches."""
+import logging
+
+from homeassistant.components.lutron_caseta import LUTRON_CASETA_DEVICES
+from homeassistant.components.lutron_caseta import LUTRON_CASETA_SMARTBRIDGE
+from homeassistant.components.lutron_caseta import LutronCasetaDevice
+from homeassistant.components.switch import SwitchDevice
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup Lutron switch."""
+    devs = []
+    for device in hass.data[LUTRON_CASETA_DEVICES]['switch']:
+        dev = LutronCasetaLight(device,
+                                hass.data[LUTRON_CASETA_SMARTBRIDGE])
+        devs.append(dev)
+
+    add_devices(devs, True)
+    return True
+
+
+class LutronCasetaLight(LutronCasetaDevice, SwitchDevice):
+    """Representation of a Lutron Caseta switch."""
+
+    def __init__(self, device, bridge):
+        """Initialize the switch."""
+        LutronCasetaDevice.__init__(self, device, bridge)
```

Comment:
If you're just calling the parent class, you don't need to overwrite the constructor.